dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtilTest.java

```diff
@@ -407,12 +409,12 @@ public void testFindImagesAndFix_doNotDeleteOtherLines() throws Exception {
     @DataProvider
     public Object[][] postTagData() {
         return new Object[][] {
-                { ":tag as builder", "newtag", ":newtag as builder"},
-                { ":tag#as builder", "newtag", ":newtag#as builder"},
-                { ":tag # comment", "newtag", ":newtag # comment"},
-                { ":tag\t# comment", "newtag", ":newtag\t# comment"},
-                { ":", "newtag", ":newtag"},
-                { ":test # :comment", "newtag", ":newtag # :comment"}
+                { ":tag as builder", "newtag", "FROM image:newtag as builder"},
+                { ":tag#as builder", "newtag", "FROM image:newtag #as builder"},
+                { ":tag # comment", "newtag", "FROM image:newtag # comment"},
+                { ":tag\t# comment", "newtag", "FROM image:newtag # comment"},
```

Comment:
It might cause the build to fail if docker only allows tabs vs. spaces for in-line comment. I'm on a very recent docker version on my current machine, it doesn't allow inline comments at all for example.

Reply:
Yeah I think if it were supported then spaces would likely be supported